python/Ganga/Core/GangaRepository/VStreamer.py

```diff
@@ -203,6 +203,8 @@ def simpleAttribute(self, node, name, value, sequence):
                 print(self.indent(), '</attribute>', file=self.out)
             else:
                 self.level += 1
+                if hasattr(value, 'accept'):
```

Comment:
This should probably be an actual type check rather than an attribute check. Furthermore, it should probably be done at the schema/descriptor level to avoid them ever being assigned rather than simply preventing them being written.